sources/platform/actors/development/programming_interface/system_events.md

```diff
@@ -5,25 +5,40 @@ slug: /actors/development/programming-interface/system-events
 sidebar_position: 4
 ---
 
+# System events in Apify Actors
+
 **Learn about system events sent to your Actor and how to benefit from them.**
 
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
 ---
 
-The system notifies Actors about various events such as a migration to another server, [abort operation](#abort-another-actor) triggered by another Actor, or the CPU being overloaded.
+## Understand system events
+
+Apify's system notifies Actors about various events, such as:
+
+- Migration to another server
+- Abort operations triggered by another Actor
+- CPU overload
+
+These events help you manage your Actor's behavior and resources effecetively.
+
+## System events
+
+The following table outlines the system events available:
 
-Currently, the system sends the following events:
 
 | Event name     | Payload | Description |
 | -------------- | ------- | ----------- |
-| `cpuInfo`      | `{ isCpuOverloaded: Boolean }` | The event is emitted approximately every second and it indicates whether the Actor is using the maximum of available CPU resources. If that's the case, the Actor should not add more workload. For example, this event is used by the AutoscaledPool class. |
-| `migrating`    | N/A | Emitted when the Actor running on the Apify platform is going to be migrated to another worker server soon. You can use it to persist the state of the Actor and abort the run, to speed up migration. For example, this is used by the RequestList class. |
-| `aborting`     | N/A | When a user aborts an Actor run on the Apify platform, they can choose to abort gracefully to allow the Actor some time before getting killed. This graceful abort emits the `aborting` event which the SDK uses to gracefully stop running crawls and you can use it to do your own cleanup as well.|
-| `persistState` | `{ isMigrating: Boolean }` | Emitted in regular intervals (by default 60 seconds) to notify all components of the Apify SDK that it is time to persist their state in order to avoid repeating all work when the Actor restarts. This event is automatically emitted together with the migrating event, in which case the `isMigrating` flag is set to `true`. Otherwise, the flag is `false`. Note that the `persistState` event is provided merely for user convenience. You can achieve the same effect using `setInterval()` and listening for the `migrating` event. |
+| `cpuInfo`      | `{ isCpuOverloaded: Boolean }` | Emitted approximately every second, indicating whether the Actor is using maximum available CPU resources. |
+| `migrating`    | N/A | Signals that the Actor will soon migrate to another worker server on the Apify platform. |
+| `aborting`     | N/A | Triggered when a user initiaties a graceful abort of an Actor run, allowing time for cleanup. |
+| `persistState` | `{ isMigrating: Boolean }` | Emitted at regular intervals  (default: _60 seconds_) to notify Apify SDK componenets to persist their state. |
```

Comment:
We should probably link to a guide on configuring the interval (you can use the `APIFY_PERSIST_STATE_INTERVAL_MILLIS` environment variable).